tests/inspector/inspectorTest.ts

```diff
@@ -186,7 +189,7 @@ class CLIMock {
     });
     this.exited = new Promise((f, r) => {
       this.process.stderr.on('data', data => {
-        r(new Error(data));
+        console.error(data.toString());
       });
       this.process.on('exit', f);
```

Comment:
Let's maybe reject if the exit code is not zero?

Reply:
done.